src/rocketData.py

```diff
@@ -313,60 +91,39 @@ def time_stamp(self):
     def setdata(self, vals):
-        lsm_ax, lsm_ay, lsm_az, lsm_mx, lsm_my, lsm_mz, lsm_gx, lsm_gy, lsm_gz, lsm_t, bme_t, bme_h, bme_p, sg_1, sg_2, sg_3, sg_4, sg_5, sg_6, sg_7, sg_8, sg_9, sg_10, sg_11, sg_12, e, ts = vals
-        print(lsm_t)
-        self.imu([lsm_ax, lsm_ay, lsm_az, lsm_mx, lsm_my, lsm_mz, lsm_gx, lsm_gy, lsm_gz, lsm_t])
-        self.bme([bme_t, bme_h, bme_p])
-        self.strain_gauges([sg_1, sg_2, sg_3, sg_4, sg_5, sg_6, sg_7, sg_8, sg_9, sg_10, sg_11, sg_12])
-        self.encoders(e)
-        self.time_stamp(ts)
-
-    def data_dict_set(self, rd):
-        self.data.update(rd)
-
-    def data_to_json(self):
-        rocketData = json.dumps(self.data, indent = 4)
-        print(rocketData)
-        return rocketData
-
-    def print_jsondata(self):
-        jsonData = self.data_to_json(self.data)
-        print(jsonData)
 
     def all_rocketdata(self):
-        alldata = [
-            self.data['lsm']['acceleration_x'],
-            self.data['lsm']['acceleration_y'],
-            self.data['lsm']['acceleration_z'],
-            self.data['lsm']['magnetometer_x'],
-            self.data['lsm']['magnetometer_y'],
-            self.data['lsm']['magnetometer_z'],
-            self.data['lsm']['gyroscope_x'],
-            self.data['lsm']['gyroscope_y'],
-            self.data['lsm']['gyroscope_z'],
-            self.data['lsm']['temperature'],
-            self.data['bme']['temperature'],
-            self.data['bme']['humidity'],
-            self.data['bme']['pressure'],
-            self.data['strain_gauges'][0],
-            self.data['strain_gauges'][1],
-            self.data['strain_gauges'][2],
-            self.data['strain_gauges'][3],
-            self.data['strain_gauges'][4],
-            self.data['strain_gauges'][5],
-            self.data['strain_gauges'][6],
-            self.data['strain_gauges'][7],
-            self.data['strain_gauges'][8],
-            self.data['strain_gauges'][9],
-            self.data['strain_gauges'][10],
-            self.data['strain_gauges'][11],
+        all_data = [
+            self.data['bme temperature'],
+            self.data['bme pressure'],
+            self.data['bme humidity'],
+            self.data['bme altitude'],
+            self.data['lsm temperature'],
+            self.data['lsm acceleration'],
+            self.data['lsm gyroscope'],
+            self.data['lsm magnetometer'],
+            self.data['adx acceleration'],
+            self.data['strain gauges'][0],
+            self.data['strain gauges'][1],
+            self.data['strain gauges'][2],
+            self.data['strain gauges'][3],
+            self.data['strain gauges'][4],
+            self.data['strain gauges'][5],
+            self.data['strain gauges'][6],
+            self.data['strain gauges'][7],
+            self.data['strain gauges'][8],
+            self.data['strain gauges'][9],
+            self.data['strain gauges'][10],
+            self.data['strain gauges'][11],
```

Comment:
```suggestion
            *self.data['strain gauges'],
```

Reply:
Not sure what this is called in Python, but in Ruby it's call the splat operator, where it inserts an array within an array.